README.md

```diff
@@ -198,6 +198,10 @@ Save the profile script, then close PowerShell and open a new PowerShell session
 Type `git fe` and then press <kbd>tab</kbd>. If posh-git has been imported, that command should tab complete to
 `git fetch`.
 
+If you want posh-git to detect your own aliases for git, then you *must* have set the alias *before* importing posh-git.
```

Comment:
I'd word this more as:

> posh-git is able to detect and provide completions for git aliases,
> which it does by looking for those aliases with `Get-Alias` when completions are set up at import time.
> This means that you must ensure your git aliases are set *before* importing posh-git.
> For example:
> ```powershell
> Set-Alias g git
> Import-Module posh-git
> ```
> will give you completions for `g`, whereas:
> ```powershell
> Import-Module posh-git
> Set-Alias g git
> ```
> will not.

Reply:
That's eerily similar to how I had originally worded it when I put it in the overview section, but then I moved it down and reworded it to fit this section better.

Reply:
I'm fine with it either way 😁